nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +255,34 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param receivers The list of receivers.
```

Comment:
Can we rename this parameter to `baseElements`?  I think that is clearer than `receivers`